randomizer/Lists/Patches.py

```diff
@@ -736,7 +736,7 @@ def setPatch(self, used):
         rotation=3492,
         group=1,
         logicregion=Regions.AngryAztecOasis,
-        logic=lambda l: l.shockwave,
+        logic=lambda l: l.vines and l.shockwave,
```

Comment:
We've decided that getting to Aztec Candy is trivial with any kong and a well timed jump so vines is unnecessary